canopen_tests/CMakeLists.txt

```diff
@@ -11,6 +11,8 @@ find_package(lely_core_libraries REQUIRED)
 
 
 generate_dcf(simple)
+generate_dcf(canopen_system)
```

Comment:
Should we add 'ros2_control' in the name - to make clearer what 'systems' means?

Reply:
We are working on a major documentation update. I think we will add a documentation on example launches.
In that case it does not matter much I suppose.